README.md

```diff
@@ -17,6 +17,15 @@ Demo live at: [demo.vercel.store](https://demo.vercel.store/)
 - Kibo Commerce Demo: https://kibocommerce.vercel.store/
 - Commerce.js Demo: https://commercejs.vercel.store/
 
+## Run minimal version locally 
+
+> deafult local provider `@vercel/commerce-local` has disabled all features (cart, auth) and use static files for the backend
```

Comment:
deafult typo.

> To run a minimal version of Next.js Commerce you can start with the default local provider...

Reply:
Install all dependencies. (There's an issue that pops up - described in the troubleshoot section) Could be useful to point that out.